backend/requirements.txt

```
@@ -0,0 +1,6 @@
+click==8.1.3
+Flask==2.2.3
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.2
+Werkzeug==2.2.3
```

Did you add these because they are dependencies of Flask?

If so we can remove them because they will be installed by the `-r` flag in the `pip install -r requirements.txt` command. 

If we define them here, then we are responsible for maintaining compatibility between them, vs relying on pip to do that for us. 

```suggestion
Flask==2.2.3
```

---

once you do the `pip freeze` command it will list all these packages because flask uses them.
ans other packages too will bring their own dependencies

---

Ok cool. I will make the changes